reports/report_builder.py

```diff
@@ -1228,6 +1230,154 @@ def ws_28(self, ws):
             self.write_overall_value(ws, value, total, overall_column, overall_row, write_overall=False)
             overall_column +=4
 
+    def ws_28b(self, ws):
+        """
+        Cols: Media; Journo Type; Sex
+        Rows: Country
+        :: Newspaper, Television, Radio, Twitter, Internet by global, region and country
```

Comment:
```suggestion
        :: Newspaper, Television, Radio, Twitter, Internet by region and country
```

Do we have by "global"?

Reply:
>Do we have by "global"?
Currently, we are generating for all 3. Should I remove the global one and only have region and country reports for this one?

Reply:
No! I meant the **output format** not _report type_. Report type should remain all 3 but format is only region and country. My understanding was that comment refers to the output format i.e. column, rows, and any other information.

Reply:
Ohh. I see now.